tests/vmi_multus_test.go

```diff
@@ -1388,31 +1424,29 @@ func validatePodKubevirtResourceName(virtClient kubecli.KubevirtClient, vmiPod *
 	return nil
 }
 
-func defaultCloudInitNetworkData() string {
-	networkData, err := libnet.CreateDefaultCloudInitNetworkData()
-	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should successfully create default cloud init network data for SRIOV")
-	return networkData
-}
-
-func cloudInitNetworkDataWithStaticIPsByMac(nicName, macAddress, ipAddress string) string {
+func cloudInitNetworkDataWithStaticIPsByMac(nicName, macAddress, ipAddress string) (string, error) {
 	networkData, err := libnet.NewNetworkData(
 		libnet.WithEthernet(nicName,
 			libnet.WithAddresses(ipAddress),
 			libnet.WithNameserverFromCluster(),
 			libnet.WithMatchingMAC(macAddress),
 		),
 	)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should successfully create static IPs by mac address cloud init network data")
-	return networkData
+	if err != nil {
+		return "", err
```

Comment:
Is the returned error contain enough information to understand where it failed?

Reply:
I think I checked it and would say yes, but will revisit